Docs/src/md/mainpage.md

```diff
@@ -0,0 +1,26 @@
+# Welcome to the AYON Cpp Dev Tools
+
+The Ayon bin bridge Client allows efficient communication with an
+[LakeFs](https://lakefs.io/) server.\n PS: we use LakeFs to distribute binary
+data across tools like Ayon-Usd addon
+
+Explore the documentation to learn more about how to use and contribute to this
+project.
+
+## Quick Links
+
+- [Getting Started Guide](md_md_Getting_Started.html)
+- [API Reference](annotated.html)
+- [Examples](md_md_Examples.html)
+### Versions
+
+> **Note** Currently older versions don't have links to newer versions because
+> of the generation order
+
+<!-- Example for adding a version link-->
+
+<!-- - [V001.01.02](../../v001.01.02/html/index.html) -->
+
+## License
+
+- This project is licensed under the Apache2.0 License.
```

Comment:
Isn't this just covered by the `LICENSE` file in the repo?
```suggestion
```

Reply:
your right probably best to write a line like : license file is in the root of the project for more in depth info's check in with Ynput team? 
just to have all grounds covered